src/components/PartnersContent/PartnersContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import { PartnersTile } from "./PartnersTile";
+import { Pagination } from "@govtechsg/tradetrust-ui-components";
+import { PartnersData, PartnerType, TypesOfPartners, PartnerDataSingle, PartnersDataProps } from "./types";
+
+const MAX_PARTNER_TILES_TO_DISPLAY = 9;
+
+export const getPartnersData = (type: PartnerType, allPartnersData: PartnerDataSingle[]): PartnersData[] => {
+  return allPartnersData
+    .filter((partner) => {
+      return partner.attributes.typeOfPartner === type;
+    })
+    .map((partner) => partner.attributes)
+    .sort((a, b) => a.companyName.localeCompare(b.companyName)); // this sort it alphabetically by companyName
+};
+
+export const PartnersContent: FunctionComponent<PartnersDataProps> = ({ partnersData }) => {
+  const [partnerState, setPartnerState] = useState<PartnerType>(TypesOfPartners.PLATFORM);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [pageData, setPageData] = useState<PartnersData[]>([]);
+  const [totalNoOfPages, setTotalNoOfPages] = useState(1);
+  const [anyPlatformData, setAnyPlatformData] = useState(false);
+  const [anyPartnersData, setAnyPartnersData] = useState(false);
```

Comment:
wouldn't `const hasPlatformData = getPartnersData(TypesOfPartners.PLATFORM, partnersData).length > 0)` work? why need to set in useEffect

Reply:
removed

Reply:
could `.some` directly with `partnersData` if all you want to know if type exists, rather than go through the whole filter, map, sort process in `getPartnersData`

Reply:
understand, but @cavacado told me to remove all the partners part and only leave the platform inside. next time then add partners section in.